recipes/pngcheck/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "4.0.0" %}
```

Comment:
This could easily be a v1 recipe and execute faster.

Reply:
honestly, until v1 gets `debug` i can't move to it. My workflow relies on debug to build me isolated environments
https://github.com/prefix-dev/rattler-build/issues/655

either
```
conda debug
```
or 
```
python build-locally.py --debug
```